.github/workflows/browser-compatibility-test.yml

```diff
@@ -0,0 +1,417 @@
+name: Browser Compatibility Test
+
+on:
+  schedule:
+    # More information on cron https://crontab.guru/
+    - cron: '0 0 * * *'
+  pull_request:
+      branches:
+        - master
```

Comment:
Temporarily adding this back again to test whether the workflow is still working. Will need to remove it before merge.

Reply:
Can we remove it now :)